charts/pulsar/values.yaml

```diff
@@ -410,6 +410,69 @@ zookeeper:
     usePolicy: true
     maxUnavailable: 1
 
+## Tiered Storage
+##
+storageOffload: {}
```

Comment:
I think this section should be moved to be under `broker`. Tiered storage is a feature of the broker, not a separate component. It will make things much clearer.

https://github.com/streamnative/charts/blob/master/charts/pulsar/values.yaml#L1019 is an example of how you can structure the values under `broker` section.